src/Arbus.Network/Abstractions/INativeHttpClient.cs

```diff
@@ -1,6 +1,10 @@
-﻿namespace Arbus.Network.Abstractions;
-
-public interface INativeHttpClient
+﻿namespace Arbus.Network.Abstractions
 {
-    Task<HttpResponseMessage> Send(HttpRequestMessage httpRequest, CancellationToken timeout, HttpCompletionOption httpCompletionOption = HttpCompletionOption.ResponseHeadersRead);
+    public interface INativeHttpClient
+    {
+        Task<HttpResponseMessage> Send(HttpRequestMessage httpRequest, CancellationToken timeout, HttpCompletionOption httpCompletionOption = HttpCompletionOption.ResponseHeadersRead);
```

Comment:
I think interface users should not use plain Send()

Reply:
Done